git-quick-stats

```diff
@@ -1083,29 +1120,38 @@ while [[ "${opt}" != "" ]]; do
         8) branchTree; showMenu;;
         9) branchesByDate; showMenu;;
        10) contributors; showMenu;;
-       11) commitsPerAuthor; showMenu;;
-       12) commitsPerDay; showMenu;;
-       13) commitsByMonth; showMenu;;
-       14) commitsByYear; showMenu;;
-       15) commitsByWeekday; showMenu;;
-       16) author=""
+       11) newDate=""
+           while [[ -z "${newDate}" ]]; do
+               read -r -p "Since what date? " newDate
+               # Test if the date provide is valid and try again if it isn't.
+               if [ ! $(date -d "${newDate}" +%s) ]; then
```

Comment:
Same comment as the previous one. Try testing `date` directly and see if that works. Reason for the change is because testing `date` directly checks success/failure immediately, suppresses unwanted output, and is more efficient than spawning a subshell.